packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -33,6 +36,9 @@ export interface AccordionProps extends UIComponentProps, ChildrenComponentProps
   /** Index of the currently active panel. */
   activeIndex?: number[] | number
 
+  /** At least one panel should be active at any time. */
+  alwaysActive?: boolean
```

Comment:
`expanded` as a name? I really don't like that we are using active, but not your fault (even in the description of the active prop we have that it means open...)